packages/web3-utils/src/index.js

```diff
@@ -328,36 +328,52 @@ var toChecksumAddress = function (address) {
  * @returns {Number} -1, 0, or 1
  */
 var compareBlockNumbers = function(a, b) {
+    // Increasing order: (genesis = earliest), safe, (finalized ~ latest), pending
```

Comment:
```suggestion
    // Increasing order: (genesis = earliest), NUMBERS or safe, (finalized ~ latest), pending
```

Reply:
This list is only for order of block tags .